docs/AboutUs.md

```diff
@@ -11,51 +11,53 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
 
 ## Project team
 
-### John Doe
+### Ken Jin
 
 <img src="images/johndoe.png" width="200px">
 
-[[homepage](http://www.comp.nus.edu.sg/~damithch)]
-[[github](https://github.com/johndoe)]
-[[portfolio](team/johndoe.md)]
+[[github](https://github.com/Fidget-Spinner/)]
+[[portfolio](team/fidget-spinner.md)]
 
-* Role: Project Advisor
+* Role: Team Lead
```

Comment:
You can put me as developer as well. Apparently an all-developer team with no leader is an actual team style in CS2103T material.

Reply:
Okay I'll do that